lucupy/plot/__init__.py

```diff
@@ -0,0 +1,98 @@
+from typing import List, Optional, Tuple
+
+import matplotlib.pyplot as plt
+import numpy as np
+import numpy.typing as npt
+
+from lucupy.minimodel.ids import ObservationID
+from lucupy.minimodel.site import Site
+
+
+def airmass(obs_airmass: npt.NDArray[float],
+            obs_id: ObservationID,
+            time_interval: Optional[npt.NDArray[int]] = None) -> None:
```

Comment:
If it's going to plot airmass, maybe we should call it something like `plot_airmass`  because the name is a bit deceptive right now.

Where will this be used?

Reply:
All these is used in GM. I thought the use case would be: 
```
import lucupy.plot as plot
plot.airmass(...)
```

Reply:
Okay, sounds good!